CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - Very basic shared-cache support, allowing multiple symbolicators to share one global cache and have faster warmup times. ([#581](https://github.com/getsentry/symbolicator/pull/581))
 - Support concurrency control and backpressure on uploading to shared cache. ([#584](https://github.com/getsentry/symbolicator/pull/584))
 - Use Google Application Credentials to authenticate to GCS in the shared cache. ([#591](https://github.com/getsentry/symbolicator/pull/591))
+- Support for `external_debug_info` section in wasm-split for external dwarf files. ([#619](https://github.com/getsentry/symbolicator/pull/619))
```

Comment:
I think this was a bad merge - at the time it was in the unreleased section but master was released shortly before merge and this section was changed to 0.4.1 but the merge still succeeded. So this change hasn't been released even though the changelog says it has